app/views/prison/visits/process_visit.html.erb

```diff
@@ -68,10 +72,25 @@
         <strong class="bold-small"><%= t('.rejection_warning') %></strong>
       </div>
     </div>
-    <%= f.submit t('.submit'), class: 'button button-primary' %>
-    <div class="push-top">
-      <%= link_to t('.email_preview'), prison_visit_email_preview_path(@visit), class: 'js-LinkPreview', target: '_blank' %>
-    </div>
+    <% if Nomis::Feature.book_to_nomis_enabled?(@visit.prison_name) %>
```

Comment:
We should stop treating our views as second class citizens and allow them to become such a mess.
This entire block should be in a decorator and be unit tested